backend/functions/src/firestore-types.ts

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable max-len */
 export type FirestoreSurveyDoc = {
 	fullName: string;
 	studentId: string;
 	identity: "a" | "b" | "c" | "d" | "e" | "f" | "g";

```

Comment:
I'm slightly worried about this though. Don't we have an option called "Others" that let students input open-ended answers? Are we then just collapsing all of those into one group?

Reply:
Good point. I was regarding this as a temporary solution, since (for now) we only have the group population as configurable. Once I fully migrate the configurations to Firestore, we could get rid of this schema.